### PR TITLE
Prepare release of SDK 1.9.1

### PR DESCRIPTION
**Pull request by bertrand-lorentz**
Only change is the removal of 3 asserts (TEDEFO-2627).

### CODE REVIEW

**Review by rousso — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# SDK 1.9.1 Release Notes
+
+This release removes 3 incorrect rules (BR-BT-00262-0211, BR-BT-00262-0212, and BR-BT-00262-0213) to allow any contract nature (works, services, and supplies) for subsidized contracts.
```
Comment (YvesJo) [inline CHANGELOG.md:3]: Removal of "constraints" would be more accurate than "incorrect rules" which applied on subsidized contracts under D24

**Review by YvesJo — COMMENTED** [review]
Ok we me. just a comment on the changelog description, however not blocking

**Review by YvesJo — DISMISSED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (ferrakaop) [reply CHANGELOG.md:3]: Agree, the rules attempted to apply an article in the directives, which turns out wasn't binding in all countries.  
So the line can be rephrased to:
"This release removes 3 rules BR-BT-00262-0211, BR-BT-00262-0212, and BR-BT-00262-0213 so there is no constraint on the nature (works, services or supplies) of subsidised contracts."

**Review by ferrakaop — CHANGES_REQUESTED** [review]
Rephrase to:
"This release removes 3 rules BR-BT-00262-0211, BR-BT-00262-0212, and BR-BT-00262-0213 so there is no constraint on the nature (works, services or supplies) of subsidised contracts."

File: CHANGELOG.md (hunk 1 above)
Comment (bertrand-lorentz) [reply CHANGELOG.md]: Rephrased as proposed.

**Review by rousso — APPROVED** [review]

**Review by YvesJo — APPROVED** [review]

**Pull request merged**